Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -123,12 +126,10 @@ namespace AZ::SceneAPI::Behaviors
         categories.emplace_back("Procedural Prefab", SceneData::PrefabGroup::TYPEINFO_Uuid());
     }
 
-    Events::ProcessingResult PrefabGroupBehavior::ExportEventHandler::UpdateManifest(
+    Events::ProcessingResult PrefabGroupBehavior::ExportEventHandler::UpdateSceneForPrefabGroup(
         Containers::Scene& scene,
-        ManifestAction action,
-        [[maybe_unused]] RequestingApplication requester)
-    {
-        // this toggle makes constructing default mesh groups and a prefab optional
+        ManifestAction action)
+    { // this toggle makes constructing default mesh groups and a prefab optional
```

Comment:
this comment does not make sense to me... what "toggle" is it referring to?

Reply:
that comment was already there, it just ended up moved. I deleted the comment.